inspirehep/modules/forms/static/js/forms/conferences_typeahead.js

```diff
@@ -50,8 +50,9 @@ define([
           url: '/api/conferences?q=conferenceautocomplete:%QUERY*',
           filter: function(response) {
             return $.map(response.hits.hits, function(el) {
-              el.metadata.city = el.metadata.address[0].cities[0];
-              el.metadata.country = el.metadata.address[0].country_code;
+              obj = el.metadata.address.find(function (obj) { return obj.hasOwnProperty('cities') });
+              el.metadata.city = obj.cities ? obj.cities[0] : '';
```

Comment:
Hmm ``obj`` is an array ([filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter)) hence ``obj.cities`` will be always ``undefined``